website/docs/AutomationProtocols.md

```diff
@@ -123,7 +121,7 @@ const { remote } = require('webdriverio')
 
```

Comment:
Should the 'We recommend wrapping your Puppeteer calls within the `call` command, so that all calls are executed before WebdriverIO continues with the next WebDriver command' paragraph be removed?